api/pb/api.pb.go

```diff
@@ -3,13 +3,14 @@
 
 package api_pb
 
-import proto "github.com/golang/protobuf/proto"
```

Comment:
Not sure why this is changing... what version of protoc are you on?

Reply:
oh interesting, didn't catch that

`libprotoc 3.9.1`

Reply:
updated to latest `libprotoc 3.11.2` and got the same

Reply:
upgraded to latest golang (1.13.5) and got yet another update here

Reply:
sorry scratch that, no change by upgrading go. same output as my current pr

Reply:
it looks like you have something different than me, carson, aaron as your commits revert it each time, 

https://github.com/textileio/go-threads/commit/5a4faf64aa08f140e83e0cd772f8f992716edea1#diff-8ac45a22aaa1306eba53a5c9ed4c8ac1

https://github.com/textileio/go-threads/commit/e3b917775f58372efe066207c0e1cb6784342bb8#diff-8ac45a22aaa1306eba53a5c9ed4c8ac1

Reply:
It's likely a version diff with `protoc-gen-go` since those live in GOPATH, not local mods. I'll make sure to `go get -u github.com/golang/protobuf/protoc-gen-go`.